client/my-sites/media-library/test/list.jsx

```diff
@@ -49,7 +49,7 @@ describe( 'MediaLibraryList item selection', () => {
 	function expectSelectedItems() {
 		defer( function () {
 			expect( mockSelectedItems ).to.have.members(
-				toArray( arguments ).map( function ( arg ) {
+				Array.from( arguments ).map( function ( arg ) {
 					return fixtures.media[ arg ];
 				} )
```

Comment:
Because of the `defer`, the `arguments` are always empty. It's a regression since #41961 where @sarayourfriend added the `defer`.

Could be fixed by using `( ...args )`, which also removes the need for `Array.from`. `...args` is a real array, not a pseudo-array like `arguments`.

Reply:
Good catch, Jarda! Fixed in 9394e754a8650e50ae4154481a837e88374322fc.